packages/@aws-cdk/aws-logs/lib/query-definition.ts

```diff
@@ -16,18 +16,21 @@ export interface QueryStringProps {
   readonly fields?: string[];
 
   /**
-  * Extracts data from a log field and creates one or more ephemeral fields that you can process further in the query.
+  * One or more statements for parsing data from a log field and creating ephemeral fields that can
+  * be processed further in the query. Each statement provided generates a separate parse line in
+  * the query string.
   *
   * @default - no parse in QueryString
   */
-  readonly parse?: string;
+  readonly parse?: string | string[];
 
   /**
-  * Filters the results of a query that's based on one or more conditions.
+  * One or more statements for filtering the results of a query based on a boolean expression. Each
+  * statement provided generates a separate filter line in the query string.
   *
   * @default - no filter in QueryString
   */
-  readonly filter?: string;
+  readonly filter?: string | string[];
```

Comment:
Same comment as above.

Reply:
I think making this one more strongly typed may be difficult, but happy to look into it if you have any suggestions :) Either way I can do the update to ensure jsii plays nicely, but will wait for your reply first.